tools/java/org/unicode/cldr/tool/GenerateCasingChart.java

```diff
@@ -101,7 +100,7 @@ public static void main(String[] args) {
             Level level = CLDRConfig.getInstance().getStandardCodes().getLocaleCoverageLevel("cldr", locale);
             boolean hasCasedLetters = changesUpper.containsSome(exemplars);
             Set<String> items = new LinkedHashSet<>();
-            CollectionUtilities.addAll(cldrFile.iterator("//ldml/contextTransforms"), items);
+            cldrFile.iterator("//ldml/contextTransforms").forEachRemaining(items::add);
```

Comment:
An example of where only an iterator is available.

Reply:
thanks for the comments, that helps.